toolchain/driver/driver.cpp

```diff
@@ -135,8 +135,8 @@ auto Driver::RunDumpSubcommand(DiagnosticConsumer& consumer,
                        .Case("llvm-ir", DumpMode::LLVMIR)
                        .Default(DumpMode::Unknown);
   if (dump_mode == DumpMode::Unknown) {
-    error_stream_ << "ERROR: Dump mode should be one of tokens, parse-tree, or "
-                     "semantics-ir.\n";
+    error_stream_ << "ERROR: Dump mode should be one of tokens, parse-tree, "
+                     "semantics-ir or llvm-ir.\n";
```

Comment:
```suggestion
                     "semantics-ir, or llvm-ir.\n";
```

We like our oxford commas. :)

Reply:
Fixed yet